cfme/physical/physical_chassis.py

```diff
@@ -0,0 +1,294 @@
+
+    ENTITY = PhysicalChassis
+
+    def all(self, provider):
+        """
+        Return all `physical_chassis` objects of the given provider.
+
+        Args:
+            provider: the provider that the physical chassis belongs to
+        """
+        physical_chassis_table = self.appliance.db.client['physical_chassis']
+        ems_table = self.appliance.db.client['ext_management_systems']
+        physical_chassis_query = (
+            self.appliance.db.client.session
+                .query(physical_chassis_table.name, physical_chassis_table.ems_ref, ems_table.name)
+                .join(ems_table, physical_chassis_table.ems_id == ems_table.id))
+
+        if self.filters.get('provider'):
+            physical_chassis_query = physical_chassis_query.filter(ems_table.name == provider.name)
+        physical_chassiss = []
+        for name, ems_ref, ems_name in physical_chassis_query.all():
+            physical_chassiss.append(self.instantiate(name=name, ems_ref=ems_ref,
+                                    provider=provider or get_crud_by_name(ems_name)))
+        return physical_chassiss
+
+
+class ComputePhysicalInfrastructureChassisView(BaseLoggedInPage):
+    """Common parts for chassis views."""
+    title = Text('.//div[@id="center_div" or @id="main-content"]//h1')
+
+    @property
+    def in_compute_physical_infrastructure_chassis(self):
+        return (self.logged_in_as_current_user and
+                self.navigation.currently_selected == ["Compute", "Physical Infrastructure",
+                                                       "Chassis"])
+
+
+class PhysicalChassisEntity(JSBaseEntity):
+    @property
+    def data(self):
+        data_dict = super(PhysicalChassisEntity, self).data
+        if 'quadicon' in data_dict and data_dict['quadicon']:
+            quad_data = document_fromstring(data_dict['quadicon'])
+            data_dict['no_host'] = int(quad_data.xpath(self.QUADRANT.format(pos="a"))[0].text)
+        return data_dict
+
+
+class PhysicalChassisDetailsToolbar(View):
+    """Represents physical chassis toolbar and its controls."""
+    identify = Dropdown(text="Identify")
+
+
+class PhysicalChassisDetailsEntities(View):
+    """Represents Details page Entities."""
+    properties = SummaryTable(title="Properties")
+    relationships = SummaryTable(title="Relationships")
+    management_network = SummaryTable(title="Management Network")
+    classis_slots = SummaryTable(title="Chassis Slots")
+
+
+class PhysicalChassisDetailsView(ComputePhysicalInfrastructureChassisView):
+    """Main PhysicalChassis details page."""
+    breadcrumb = BreadCrumb()
+    toolbar = View.nested(PhysicalChassisDetailsToolbar)
+    entities = View.nested(PhysicalChassisDetailsEntities)
+
+    @property
+    def is_displayed(self):
+        title = "{name} (Summary)".format(name=self.context["object"].name)
+        return (self.in_compute_physical_infrastructure_chassis and
```

Comment:
As mentioned in the comment above, we could refer to `is_displayed` of the parent via `super(self.__class__, self).is_displayed` instead implementing custom function names.